account-kit/core/src/types.ts

```diff
@@ -78,7 +78,7 @@ type RpcConnectionConfig =
       chain: Chain;
       chains: {
         chain: Chain;
-        policyId?: string;
+        policyId?: string | string[];
```

Comment:
can one chain have multiple policy ids?

Reply:
Yes, with the idea being that when we have gas grants you can add both the grant as well as your own policy. In general this model might be worth revisiting now that we also have multi-chain policices.

Reply:
yea with multi-chain policies, it almost feels like we can add support for that at the top level and just have a `policyId?: string[]` and start pushing folks towards that